bestofjs/bestofjs
# Upgrade Radix deps

## Goal

- Upgrade Radix dependencies to fix warning in the console, from the dropdown menus
- Fix warning when running `pnpm install` by upgrading `swr` and `msw` in `bestof-webui` app (the "classic" SPA)
- Upgrade static API root URL in `bestofjs-webui`

## How to test

Ensure no error shows up in the console, when running the top page

```
Warning: Accessing element.ref is no longer supported. ref is now a regular prop. It will be removed from the JSX Element type in a future release.
```

## Screenshots
